src/scripts/bootstrap_trigger/bootstrap_trigger.py

```diff
@@ -0,0 +1,176 @@
+"""
+Submit Synapse files stored on an external S3 bucket to a Glue workflow.
+Of course, this assumes that the workflow has read permissions on the S3
+bucket.
+
+The --diff-* arguments allow a parquet dataset in a Synapse-linked S3 location to
+be diffed upon before submitting matching --query results to the --glue-workflow.
+In this way, data which has already been processed by the pipeline is not
+processed again.
+"""
+import json
+import argparse
+import boto3
+import synapseclient
+from pyarrow import fs, parquet
+
+
+def read_args():
+    parser = argparse.ArgumentParser(
+            description=("Submit files on Synapse to an AWS --glue-workflow"))
+    parser.add_argument("--file-view",
+                        required=True,
+                        help=("The Synapse ID of a file view containing "
+                              "files to be submitted."))
+    parser.add_argument("--raw-folder-id",
```

Comment:
What is the folder structure?  How do you have to deal with nested folder structure?

Reply:
The folder structure doesn't matter. We only use this value to get an STS token. The file entity interfacing happens on a file view.